docs/docs/external/getting-started/test-network/setup-local.md

```diff
@@ -186,7 +195,7 @@ Build the generic (i.e., common to all DLTs) relay module as follows:
   ```bash
   make
   ```
-  
+
 ##### Deployment
```

Comment:
Can we remove `Running Relay in Host` heading, and increase the heading size of `Building` and  `Deployment`?
Same for Fabric-driver.

Reply:
Fixed